tests/e2e/README.md

```diff
@@ -34,10 +34,14 @@ Download the latest version (currently 2.2.19) of Vagrant [*from the website*](h
 
 ### Libvirt
 Follow the OS specific guides to install libvirt/qemu on your host:  
-    - [openSUSE](https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-vt-installation.html)
-    - [ubuntu](https://ubuntu.com/server/docs/virtualization-libvirt)
-    - [debian](https://wiki.debian.org/KVM#Installation)
-    - [fedora](https://developer.fedoraproject.org/tools/virtualization/installing-libvirt-and-virt-install-on-fedora-linux.html)
+- [openSUSE](https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-vt-installation.html)  
+- [ubuntu 20.04](https://ubuntu.com/server/docs/virtualization-libvirt)  
+- ubuntu 22.04: 
```

Comment:
Repeated?
```suggestion
```

Reply:
I gave each ubuntu version a separate line. For ubuntu 20.04 the offical docs seem to work fine, but not for the latest LTS.

Reply:
oh I see. I missed that the line above was 20.04 not 22.04